test/tests/Pathfinding.cpp

```diff
@@ -117,7 +117,7 @@ class PathfindingTestBase : public testing::Test
 
             // Check that the peep is still on a footpath. Use next_z instead of pos->z here because pos->z will change
             // when the peep is halfway up a slope, but next_z will not change until they move to the next tile.
-            EXPECT_NE(map_get_footpath_element(TileCoordsXYZ{ pos->x, pos->y, peep->next_z }.ToCoordsXYZ()), nullptr);
+            EXPECT_NE(map_get_footpath_element({ TileCoordsXY{ pos->x, pos->y }.ToCoordsXY(), peep->NextLoc.z }), nullptr);
```

Comment:
I think that

```suggestion
            EXPECT_NE(map_get_footpath_element({ pos.ToCoordsXY(), peep->NextLoc.z }), nullptr);
```

should work.

Reply:
Yes that would work not sure what I was thinking whilst doing this.